app/pods/bucket-type/model.js

```diff
@@ -28,6 +28,23 @@ var BucketType = DS.Model.extend({
     bucketList: DS.belongsTo('bucket-list'),
 
     /**
+     * Bucket Type Properties object.
```

Comment:
I mentioned it the bullet points, but I reordered all models so relationships are first, attributes second, methods third, and observable methods last. I personally think its easier to scan a model that way. I probably undid some of the alphabetizing when I did that, so I can at least make sure they are alphabetized within their given order. I'll go ahead and do that before merging.